packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache

```diff
@@ -59,7 +59,7 @@
         </div>
 
       <div class="flex justify-between">
-        <a href="/" id="use-different" class="link-blue" data-flow-event="use-different-account">{{#t}}Use a different account{{/t}}</a>
+        <a href="/" id="use-different" class="link-blue mr-2" data-flow-event="use-different-account">{{#t}}Use a different account{{/t}}</a>
```

Comment:
Q: Is there a difference between `mr-2` on one of the links versus something like `gap-2` on the parent flex `<div>`?

Class | Properties
:----|:----
[`mr-2`](https://tailwindcss.com/docs/margin#class-reference) | `margin-right: 0.5rem; /* 8px */`
[`gap-2`](https://tailwindcss.com/docs/gap#class-reference) | `gap: 0.5rem; /* 8px */`